Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

```diff
@@ -13,14 +13,40 @@ Microsoft Defender Advanced Threat Protection Get Machine Action Status
 2. Managing machines and performing actions on them.
 3. Blocking files and applications.
 
-## Configure Microsoft Defender Advanced Threat Protection on Cortex XSOAR
+## Required Permissions
+##### Microsoft Graph Permissions:
+* Directory.Read.All - Delegated
+* ThreatIndicators.Read.All - Application
+* ThreatIndicators.ReadWrite.OwnedBy - Application
+##### WindowsDefenderATP Permissions:
+* AdvancedQuery.Read.All - Application
+* Alert.Read.All - Application
+* Alert.ReadWrite.All - Application
```

Comment:
this permission includes the previous permission, why do we need both?

Reply:
We don't, I took it from our app registrations. I'll go through Microsoft API reference and split the permissions for each command.